### ISSUE TITLE
Text File

### ISSUE THREAD

**Issue by NozomiRem** [AerialX/pso2-go#1]
How can I rebuild the Pso2 text files?
With NIFL v0.05 performing the command
Nifl.exe ui_mainmode.text> ui_mainmode.txt
Information can be extracted
Is it possible to reverse that process?
Return the txt a Text?

**Comment by AerialX** [comment]
No idea what nifl.exe is. The code in this repo can be used to repack text/nifl files fairly easily, but there is no tool for that purpose included.

**Comment by NozomiRem** [comment]
But would it be possible to translate the game into another language with
this?

2017-02-27 14:06 GMT-03:00 Aaron Lindsay <notifications@github.com>:

**Comment by AerialX** [comment]
Certainly, that's why the code was written; to power the english translation. It's however not packaged into a tool a non-programmer could use.

For example, [cmd/pso2-text](https://github.com/AerialX/pso2-go/blob/master/cmd/pso2-text/text.go) when used with the `-p -w` flags will write out a new text file with every string replaced with `LOLOLOL`. Not very useful, but it's an example that can be adapted to replace strings from a translation document / lookup table instead.

**Comment by NozomiRem** [comment]
My intention is to try to translate the game into our native language
But using wxMEdit to edit the text / file
This limits number of slides

**Comment by NozomiRem** [comment]
Excuse me if you do not quite understand what I mean
My English is not very good

**Comment by NozomiRem** [comment]
tmp/sandbox795523324/main.go:7:2: cannot find package "aaronlindsay.com/go/pkg/pso2/text" in any of:
	/usr/local/go/src/aaronlindsay.com/go/pkg/pso2/text (from $GOROOT)
	/go/src/aaronlindsay.com/go/pkg/pso2/text (from $GOPATH)

**Comment by NozomiRem** [comment]
import (
	"fmt"
	"os"
	"flag"
	"aaronlindsay.com/go/pkg/pso2/text"
)

"Aaronlindsay.com/go/pkg/pso2/text" it seems that the error goes by there should I change it?

**Comment by AerialX** [comment]
The go dependencies are annoying to set up. You have to do something like:

    export GOPATH="$HOME/.go/"
    mkdir -p "$HOME/.go/src/aaronlindsay.com/go/pkg"
    ln -s /your/path/to/cloned/pso2-go "$HOME/.go/src/aaronlindsay.com/go/pkg/pso2"

(this is from memory and may not be exactly correct, and slightly different on Windows if you're not using the unix subsystem)

**Comment by WeiTen** [comment]
Hello I am sorry for my bad english but I am learning. Im a Chinese player and want to take apart the PSO2 .text file and turn it into normal .txt file and then edit and then back to .text file.

Can you help me with this?

I am beginning coder level but learn.

Thank you.
